src/components/portrait/portrait.config.ts

```diff
@@ -1,6 +1,19 @@
 export const PORTRAIT_SHAPES = ["circle", "square"];
-export const PORTRAIT_SIZES = ["XS", "S", "M", "ML", "L", "XL", "XXL"];
-export const PORTRAIT_SIZE_PARAMS = {
+type PortraitSize = "XS" | "S" | "M" | "ML" | "L" | "XL" | "XXL";
+export const PORTRAIT_SIZES: PortraitSize[] = [
```

Comment:
**comment** (non-blocking): I don't mind what you've done here - there's no harm in making explicit types for these, especially if we end up wanting to export them later. But you didn't actually need to do it to fix the type errors you get in your Playwright stuff if you remove them - just adding `as const` to both `PORTRAIT_SIZES` and `PORTRAIT_SIZE_PARAMS` is enough. I'm not saying this is better though, it's just an alternative.

Reply:
Yeah fair, I think I'll still leave this in anyways, as I don't think there's any harm in adding some types, especially as they might be needed further down the line